time_stamper_cv/src/led_parser.cpp

```diff
@@ -0,0 +1,92 @@
+#include "led_parser.h"
+
+#include <utility>
+
+LedParser::LedParser(LedRowConfig led_row_config, int image_crop_size)
```

Comment:
hmm LedStateDetector would be a more verbose name

Reply:
Renamed to LedStateParser to not confuse it with KeypointDetector